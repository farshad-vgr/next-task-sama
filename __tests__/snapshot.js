import { render } from "@testing-library/react";

import {
	SubmitButton,
	ConfirmButton,
	BookingButton,
	CloseButton,
	MenuButton,
	Asidebar,
	SearchButton,
	SearchInput,
	SignupInput,
	ConfirmInput,
	ConfirmForm,
	JumperButton,
	GeneratePdfButton,
} from "../components/index.ts";

describe("Snapshot Testing", () => {
	it("Renderes SubmitButton unchanged?", () => {
		const { container } = render(<SubmitButton btnText="ورود" isCaptchaValid={false} />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes ConfirmButton unchanged?", () => {
		const { container } = render(<ConfirmButton btnText="تایید" isDisable={true} />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes BookingButton unchanged?", () => {
		const { container } = render(<BookingButton btnText="ثبت نوبت" isDisable={true} />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes GeneratePdfButton unchanged?", () => {
		const { container } = render(<GeneratePdfButton btnText="دانلود فرم" isDisable={true} />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes CloseButton unchanged?", () => {
		const { container } = render(<CloseButton />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes MenuButton unchanged?", () => {
		const { container } = render(<MenuButton />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes Asidebar unchanged?", () => {
		const { container } = render(<Asidebar isAside={false} />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes SearchButton unchanged?", () => {
		const { container } = render(<SearchButton />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes SearchInput unchanged?", () => {
		const { container } = render(<SearchInput />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes SignupInput unchanged?", () => {
		const { container } = render(<SignupInput />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes ConfirmInput unchanged?", () => {
		const { container } = render(<ConfirmInput />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes ConfirmForm unchanged?", () => {
		const { container } = render(<ConfirmForm />);
		expect(container).toMatchSnapshot();
	});

	it("Renderes JumperButton unchanged?", () => {
		const { container } = render(<JumperButton />);
		expect(container).toMatchSnapshot();
	});
});
