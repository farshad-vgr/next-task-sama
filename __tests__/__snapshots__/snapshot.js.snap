// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing Renderes Asidebar unchanged? 1`] = `
<div>
  <div
    class="absolute top-0 -right-full w-full h-full bg-gray-900 bg-opacity-50 transition-all duration-500 z-10"
    data-testid="asidebar-backdrop"
  />
  <aside
    class="absolute top-0 -left-60 w-60 h-full bg-gray-900 bg-opacity-80 transition-all duration-700 z-10"
    data-testid="asidebar-container"
  >
    <button
      class="flex justify-center items-center absolute top-3 left-5 w-7 h-7 mt-1 rounded-full text-base text-red-600 border border-red-600 shadow-[0_0_0.25rem_1px_inset] shadow-red-500"
      data-testid="close-button"
      type="button"
    >
      <svg
        class="w-5 h-5"
        data-testid="close-button-svg"
        fill="none"
        stroke="currentColor"
        stroke-width="3"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M6 18L18 6M6 6l12 12"
          stroke-linecap="round"
          stroke-linejoin="round"
        />
      </svg>
    </button>
    <div
      class="absolute top-52 left-4"
      data-testid="asidebar-search-button-wrapper"
    >
      <button
        class="w-10 min-h-[2.5rem] p-1 rounded-[1.25rem] text-base bg-red-700 text-white hover:bg-red-600 hover:cursor-pointer transition-all duration-500"
        data-testid="search-button"
        type="button"
      >
        <svg
          class="absolute top-2 left-2 w-6 h-6"
          data-testid="search-button-svg"
          fill="none"
          stroke="currentColor"
          stroke-width="3"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
            stroke-linecap="round"
            stroke-linejoin="round"
          />
        </svg>
      </button>
    </div>
    <section
      class="mt-12 p-4"
      data-testid="asidebar-navbar-wrapper"
    >
      <nav>
        <ul
          class="flex flex-col gap-5 sm:gap-10 md:gap-16 text-md text-slate-200 sm:text-slate-600"
        >
          <li
            class="text-lg hover:text-red-700 hover:underline"
          >
            <a
              href="/"
            >
              <span
                class="flex gap-1"
              >
                <svg
                  class="w-5 h-5"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M2.25 12l8.954-8.955c.44-.439 1.152-.439 1.591 0L21.75 12M4.5 9.75v10.125c0 .621.504 1.125 1.125 1.125H9.75v-4.875c0-.621.504-1.125 1.125-1.125h2.25c.621 0 1.125.504 1.125 1.125V21h4.125c.621 0 1.125-.504 1.125-1.125V9.75M8.25 21h8.25"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                  <path
                    d=""
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                ثبت نام
              </span>
            </a>
          </li>
          <li
            class="text-lg hover:text-red-700 hover:underline"
          >
            <a
              href="/confirmation"
            >
              <span
                class="flex gap-1"
              >
                <svg
                  class="w-5 h-5"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8.625 9.75a.375.375 0 11-.75 0 .375.375 0 01.75 0zm0 0H8.25m4.125 0a.375.375 0 11-.75 0 .375.375 0 01.75 0zm0 0H12m4.125 0a.375.375 0 11-.75 0 .375.375 0 01.75 0zm0 0h-.375m-13.5 3.01c0 1.6 1.123 2.994 2.707 3.227 1.087.16 2.185.283 3.293.369V21l4.184-4.183a1.14 1.14 0 01.778-.332 48.294 48.294 0 005.83-.498c1.585-.233 2.708-1.626 2.708-3.228V6.741c0-1.602-1.123-2.995-2.707-3.228A48.394 48.394 0 0012 3c-2.392 0-4.744.175-7.043.513C3.373 3.746 2.25 5.14 2.25 6.741v6.018z"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                  <path
                    d=""
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                کد تایید
              </span>
            </a>
          </li>
          <li
            class="text-lg hover:text-red-700 hover:underline"
          >
            <a
              href="/booking"
            >
              <span
                class="flex gap-1"
              >
                <svg
                  class="w-5 h-5"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M11.35 3.836c-.065.21-.1.433-.1.664 0 .414.336.75.75.75h4.5a.75.75 0 00.75-.75 2.25 2.25 0 00-.1-.664m-5.8 0A2.251 2.251 0 0113.5 2.25H15c1.012 0 1.867.668 2.15 1.586m-5.8 0c-.376.023-.75.05-1.124.08C9.095 4.01 8.25 4.973 8.25 6.108V8.25m8.9-4.414c.376.023.75.05 1.124.08 1.131.094 1.976 1.057 1.976 2.192V16.5A2.25 2.25 0 0118 18.75h-2.25m-7.5-10.5H4.875c-.621 0-1.125.504-1.125 1.125v11.25c0 .621.504 1.125 1.125 1.125h9.75c.621 0 1.125-.504 1.125-1.125V18.75m-7.5-10.5h6.375c.621 0 1.125.504 1.125 1.125v9.375m-8.25-3l1.5 1.5 3-3.75"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                  <path
                    d=""
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                نوبت دهی
              </span>
            </a>
          </li>
        </ul>
      </nav>
    </section>
  </aside>
</div>
`;

exports[`Snapshot Testing Renderes BookingButton unchanged? 1`] = `
<div>
  <button
    class="flex justify-center items-center self-center gap-1 max-w-fit px-3 outline-none rounded-lg booking-button cursor-not-allowed text-black bg-slate-500 opacity-50"
    data-testid="booking-button"
    disabled=""
    type="submit"
  >
    <svg
      class="w-6 h-6"
      data-testid="booking-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="1.5"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M19 7.5v3m0 0v3m0-3h3m-3 0h-3m-2.25-4.125a3.375 3.375 0 11-6.75 0 3.375 3.375 0 016.75 0zM4 19.235v-.11a6.375 6.375 0 0112.75 0v.109A12.318 12.318 0 0110.374 21c-2.331 0-4.512-.645-6.374-1.766z"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
    <span
      class="mt-2"
      data-testid="booking-button-span"
    >
      ثبت نوبت
    </span>
  </button>
</div>
`;

exports[`Snapshot Testing Renderes BookingGender unchanged? 1`] = `
<div>
  <section>
    <div
      class="relative flex justify-start w-full overflow-hidden"
      data-testid="booking-gender-container"
    >
      <label
        class="me-2"
        data-testid="booking-gender-title"
      >
        جنسیت:
      </label>
      <input
        checked=""
        class="me-1 mb-1 accent-red-700"
        data-testid="booking-gender-input-one"
        id="male"
        name="gender"
        type="radio"
        value="male"
      />
      <label
        class="me-4"
        data-testid="booking-gender-label-one"
        for="male"
      >
        مرد
      </label>
      <input
        class="me-1 mb-1 accent-red-700"
        data-testid="booking-gender-input-two"
        id="female"
        name="gender"
        type="radio"
        value="female"
      />
      <label
        data-testid="booking-gender-label-two"
        for="female"
      >
        زن
      </label>
    </div>
  </section>
</div>
`;

exports[`Snapshot Testing Renderes BookingInsurance unchanged? 1`] = `
<div>
  <section>
    <div
      class="relative flex justify-start w-full overflow-hidden"
      data-testid="booking-insurance-container"
    >
      <label
        class="me-2"
        data-testid="booking-insurance-title"
      >
        بیمه درمانی:
      </label>
      <input
        checked=""
        class="me-1 mb-2 accent-red-700"
        data-testid="booking-insurance-input-one"
        id="false"
        name="insurance"
        type="radio"
        value="false"
      />
      <label
        class="me-4"
        data-testid="booking-insurance-label-one"
        for="false"
      >
        ندارد
      </label>
      <input
        class="me-1 mb-2 accent-red-700"
        data-testid="booking-insurance-input-two"
        id="true"
        name="insurance"
        type="radio"
        value="true"
      />
      <label
        data-testid="booking-insurance-label-two"
        for="true"
      >
        دارد
      </label>
    </div>
  </section>
</div>
`;

exports[`Snapshot Testing Renderes CloseButton unchanged? 1`] = `
<div>
  <button
    class="flex justify-center items-center absolute top-3 left-5 w-7 h-7 mt-1 rounded-full text-base text-red-600 border border-red-600 shadow-[0_0_0.25rem_1px_inset] shadow-red-500"
    data-testid="close-button"
    type="button"
  >
    <svg
      class="w-5 h-5"
      data-testid="close-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="3"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M6 18L18 6M6 6l12 12"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
  </button>
</div>
`;

exports[`Snapshot Testing Renderes ConfirmButton unchanged? 1`] = `
<div>
  <button
    class="flex justify-center items-center self-center gap-1 max-w-fit px-3 outline-none rounded-lg cursor-not-allowed text-black bg-slate-500 opacity-50"
    data-testid="confirm-button"
    disabled=""
    type="submit"
  >
    <svg
      class="w-6 h-6"
      data-testid="confirm-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="1.5"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M9 12.75L11.25 15 15 9.75M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
    <span
      class="mt-2"
      data-testid="confirm-button-span"
    >
      تایید
    </span>
  </button>
</div>
`;

exports[`Snapshot Testing Renderes ConfirmForm unchanged? 1`] = `
<div>
  <form
    class="flex flex-col justify-center gap-4 w-full"
    data-testid="confirm-form"
  >
    <section>
      <div
        class="relative flex justify-center w-full overflow-hidden signup-input confirm-input"
      >
        <div
          class="flex justify-center items-center absolute right-0 w-10 h-10 leading-8 outline-none text-white bg-red-700 border-b-[3px] border-b-red-700 rounded-[0_0.5rem_0_0]"
        >
          <svg
            class="w-6 h-6"
            data-testid="confirm-input-svg"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16.862 4.487l1.687-1.688a1.875 1.875 0 112.652 2.652L6.832 19.82a4.5 4.5 0 01-1.897 1.13l-2.685.8.8-2.685a4.5 4.5 0 011.13-1.897L16.863 4.487zm0 0L19.5 7.125"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </div>
        <input
          class="py-2 px-4 pr-12 pb-0 w-full h-10 outline-none text-sm border-b-[3px] border-b-red-700 rounded-[0.5rem_0.5rem_0_0]"
          data-testid="confirm-input"
          maxlength="15"
          minlength="3"
          placeholder="کد پیامک شده را وارد کنید"
          required=""
          type="number"
          value=""
        />
      </div>
    </section>
    <button
      class="flex justify-center items-center self-center gap-1 max-w-fit px-3 outline-none rounded-lg cursor-not-allowed text-black bg-slate-500 opacity-50"
      data-testid="confirm-button"
      disabled=""
      type="submit"
    >
      <svg
        class="w-6 h-6"
        data-testid="confirm-button-svg"
        fill="none"
        stroke="currentColor"
        stroke-width="1.5"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9 12.75L11.25 15 15 9.75M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
          stroke-linecap="round"
          stroke-linejoin="round"
        />
      </svg>
      <span
        class="mt-2"
        data-testid="confirm-button-span"
      >
        تایید
      </span>
    </button>
  </form>
</div>
`;

exports[`Snapshot Testing Renderes ConfirmInput unchanged? 1`] = `
<div>
  <section>
    <div
      class="relative flex justify-center w-full overflow-hidden signup-input confirm-input"
    >
      <div
        class="flex justify-center items-center absolute right-0 w-10 h-10 leading-8 outline-none text-white bg-red-700 border-b-[3px] border-b-red-700 rounded-[0_0.5rem_0_0]"
      >
        <svg
          class="w-6 h-6"
          data-testid="confirm-input-svg"
          fill="none"
          stroke="currentColor"
          stroke-width="1.5"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16.862 4.487l1.687-1.688a1.875 1.875 0 112.652 2.652L6.832 19.82a4.5 4.5 0 01-1.897 1.13l-2.685.8.8-2.685a4.5 4.5 0 011.13-1.897L16.863 4.487zm0 0L19.5 7.125"
            stroke-linecap="round"
            stroke-linejoin="round"
          />
        </svg>
      </div>
      <input
        class="py-2 px-4 pr-12 pb-0 w-full h-10 outline-none text-sm border-b-[3px] border-b-red-700 rounded-[0.5rem_0.5rem_0_0]"
        data-testid="confirm-input"
        maxlength="15"
        minlength="3"
        placeholder="کد پیامک شده را وارد کنید"
        required=""
        type="number"
        value=""
      />
    </div>
  </section>
</div>
`;

exports[`Snapshot Testing Renderes GeneratePdfButton unchanged? 1`] = `
<div>
  <button
    class="flex justify-center items-center self-center gap-1 max-w-fit px-3 outline-none rounded-lg download-button cursor-not-allowed text-black bg-slate-500 opacity-50"
    data-testid="generate-pdf-button"
    disabled=""
    type="button"
  >
    <svg
      class="w-6 h-6"
      data-testid="generate-pdf-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="1.5"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M19.5 14.25v-2.625a3.375 3.375 0 00-3.375-3.375h-1.5A1.125 1.125 0 0113.5 7.125v-1.5a3.375 3.375 0 00-3.375-3.375H8.25m.75 12l3 3m0 0l3-3m-3 3v-6m-1.5-9H5.625c-.621 0-1.125.504-1.125 1.125v17.25c0 .621.504 1.125 1.125 1.125h12.75c.621 0 1.125-.504 1.125-1.125V11.25a9 9 0 00-9-9z"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
    <span
      class="mt-2"
      data-testid="generate-pdf-button-span"
    >
      دانلود فرم
    </span>
  </button>
</div>
`;

exports[`Snapshot Testing Renderes JumperButton unchanged? 1`] = `
<div>
  <button
    class="fixed bottom-3 right-3 p-1 text-undefined-500 bg-black bg-opacity-20 border-2 border-dashed rounded-full border-undefined-500 hover:bg-opacity-30 transition-all duration-300"
    data-testid="jumper-button"
    title="برو به بالای صفحه!"
    type="button"
  >
    <svg
      class="w-6 h-6"
      data-testid="jumper-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="2"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M4.5 12.75l7.5-7.5 7.5 7.5m-15 6l7.5-7.5 7.5 7.5"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
  </button>
</div>
`;

exports[`Snapshot Testing Renderes MenuButton unchanged? 1`] = `
<div>
  <button
    class="flex justify-center items-center absolute top-2 left-5 w-10 h-10 mt-1 rounded-full text-base bg-red-700 text-white hover:cursor-pointer"
    data-testid="menu-button"
    type="button"
  >
    <svg
      class="w-6 h-6"
      data-testid="menu-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="3"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
  </button>
  <div
    class="absolute top-0 -right-full w-full h-full bg-gray-900 bg-opacity-50 transition-all duration-500 z-10"
    data-testid="asidebar-backdrop"
  />
  <aside
    class="absolute top-0 -left-60 w-60 h-full bg-gray-900 bg-opacity-80 transition-all duration-700 z-10"
    data-testid="asidebar-container"
  >
    <button
      class="flex justify-center items-center absolute top-3 left-5 w-7 h-7 mt-1 rounded-full text-base text-red-600 border border-red-600 shadow-[0_0_0.25rem_1px_inset] shadow-red-500"
      data-testid="close-button"
      type="button"
    >
      <svg
        class="w-5 h-5"
        data-testid="close-button-svg"
        fill="none"
        stroke="currentColor"
        stroke-width="3"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M6 18L18 6M6 6l12 12"
          stroke-linecap="round"
          stroke-linejoin="round"
        />
      </svg>
    </button>
    <div
      class="absolute top-52 left-4"
      data-testid="asidebar-search-button-wrapper"
    >
      <button
        class="w-10 min-h-[2.5rem] p-1 rounded-[1.25rem] text-base bg-red-700 text-white hover:bg-red-600 hover:cursor-pointer transition-all duration-500"
        data-testid="search-button"
        type="button"
      >
        <svg
          class="absolute top-2 left-2 w-6 h-6"
          data-testid="search-button-svg"
          fill="none"
          stroke="currentColor"
          stroke-width="3"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
            stroke-linecap="round"
            stroke-linejoin="round"
          />
        </svg>
      </button>
    </div>
    <section
      class="mt-12 p-4"
      data-testid="asidebar-navbar-wrapper"
    >
      <nav>
        <ul
          class="flex flex-col gap-5 sm:gap-10 md:gap-16 text-md text-slate-200 sm:text-slate-600"
        >
          <li
            class="text-lg hover:text-red-700 hover:underline"
          >
            <a
              href="/"
            >
              <span
                class="flex gap-1"
              >
                <svg
                  class="w-5 h-5"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M2.25 12l8.954-8.955c.44-.439 1.152-.439 1.591 0L21.75 12M4.5 9.75v10.125c0 .621.504 1.125 1.125 1.125H9.75v-4.875c0-.621.504-1.125 1.125-1.125h2.25c.621 0 1.125.504 1.125 1.125V21h4.125c.621 0 1.125-.504 1.125-1.125V9.75M8.25 21h8.25"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                  <path
                    d=""
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                ثبت نام
              </span>
            </a>
          </li>
          <li
            class="text-lg hover:text-red-700 hover:underline"
          >
            <a
              href="/confirmation"
            >
              <span
                class="flex gap-1"
              >
                <svg
                  class="w-5 h-5"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8.625 9.75a.375.375 0 11-.75 0 .375.375 0 01.75 0zm0 0H8.25m4.125 0a.375.375 0 11-.75 0 .375.375 0 01.75 0zm0 0H12m4.125 0a.375.375 0 11-.75 0 .375.375 0 01.75 0zm0 0h-.375m-13.5 3.01c0 1.6 1.123 2.994 2.707 3.227 1.087.16 2.185.283 3.293.369V21l4.184-4.183a1.14 1.14 0 01.778-.332 48.294 48.294 0 005.83-.498c1.585-.233 2.708-1.626 2.708-3.228V6.741c0-1.602-1.123-2.995-2.707-3.228A48.394 48.394 0 0012 3c-2.392 0-4.744.175-7.043.513C3.373 3.746 2.25 5.14 2.25 6.741v6.018z"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                  <path
                    d=""
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                کد تایید
              </span>
            </a>
          </li>
          <li
            class="text-lg hover:text-red-700 hover:underline"
          >
            <a
              href="/booking"
            >
              <span
                class="flex gap-1"
              >
                <svg
                  class="w-5 h-5"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M11.35 3.836c-.065.21-.1.433-.1.664 0 .414.336.75.75.75h4.5a.75.75 0 00.75-.75 2.25 2.25 0 00-.1-.664m-5.8 0A2.251 2.251 0 0113.5 2.25H15c1.012 0 1.867.668 2.15 1.586m-5.8 0c-.376.023-.75.05-1.124.08C9.095 4.01 8.25 4.973 8.25 6.108V8.25m8.9-4.414c.376.023.75.05 1.124.08 1.131.094 1.976 1.057 1.976 2.192V16.5A2.25 2.25 0 0118 18.75h-2.25m-7.5-10.5H4.875c-.621 0-1.125.504-1.125 1.125v11.25c0 .621.504 1.125 1.125 1.125h9.75c.621 0 1.125-.504 1.125-1.125V18.75m-7.5-10.5h6.375c.621 0 1.125.504 1.125 1.125v9.375m-8.25-3l1.5 1.5 3-3.75"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                  <path
                    d=""
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                نوبت دهی
              </span>
            </a>
          </li>
        </ul>
      </nav>
    </section>
  </aside>
</div>
`;

exports[`Snapshot Testing Renderes SearchButton unchanged? 1`] = `
<div>
  <button
    class="w-10 min-h-[2.5rem] p-1 rounded-[1.25rem] text-base bg-red-700 text-white hover:bg-red-600 hover:cursor-pointer transition-all duration-500"
    data-testid="search-button"
    type="button"
  >
    <svg
      class="absolute top-2 left-2 w-6 h-6"
      data-testid="search-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="3"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
  </button>
</div>
`;

exports[`Snapshot Testing Renderes SearchInput unchanged? 1`] = `
<div>
  <input
    class="absolute top-1 right-2 ml-2 -mr-1 p-1 pb-0 pr-3 w-[80%] sm:w-[88%] h-[2rem] rounded-full outline-none text-black"
    data-testid="search-input"
    maxlength="15"
    placeholder="جستجو..."
    type="text"
    value=""
  />
</div>
`;

exports[`Snapshot Testing Renderes SignupInput unchanged? 1`] = `
<div>
  <section>
    <div
      class="relative flex justify-center w-full overflow-hidden signup-input confirm-input"
    >
      <div
        class="flex justify-center items-center absolute right-0 w-10 h-10 leading-8 outline-none text-white bg-red-700 border-b-[3px] border-b-red-700 rounded-[0_0.5rem_0_0]"
      >
        <svg
          class="w-6 h-6"
          data-testid="signup-input-svg"
          fill="none"
          stroke="currentColor"
          stroke-width="1.5"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16.862 4.487l1.687-1.688a1.875 1.875 0 112.652 2.652L6.832 19.82a4.5 4.5 0 01-1.897 1.13l-2.685.8.8-2.685a4.5 4.5 0 011.13-1.897L16.863 4.487zm0 0L19.5 7.125"
            stroke-linecap="round"
            stroke-linejoin="round"
          />
        </svg>
      </div>
      <input
        class="py-2 px-4 pr-12 pb-0 w-full h-10 outline-none text-sm border-b-[3px] border-b-red-700 rounded-[0.5rem_0.5rem_0_0]"
        data-testid="signup-input"
        maxlength="15"
        minlength="3"
        placeholder="کدملی / شماره موبایل / شماره قبض"
        required=""
        type="number"
        value=""
      />
    </div>
  </section>
</div>
`;

exports[`Snapshot Testing Renderes SubmitButton unchanged? 1`] = `
<div>
  <button
    class="flex justify-center items-center self-center gap-1 max-w-fit px-3 py-[2px] outline-none rounded-lg cursor-not-allowed text-black bg-slate-500 opacity-50"
    data-testid="submit-button"
    disabled=""
    type="submit"
  >
    <svg
      class="w-6 h-6"
      data-testid="submit-button-svg"
      fill="none"
      stroke="currentColor"
      stroke-width="1.5"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M15.75 9V5.25A2.25 2.25 0 0013.5 3h-6a2.25 2.25 0 00-2.25 2.25v13.5A2.25 2.25 0 007.5 21h6a2.25 2.25 0 002.25-2.25V15M12 9l-3 3m0 0l3 3m-3-3h12.75"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
    <span
      class="mt-1"
      data-testid="submit-button-span"
    >
      ورود
    </span>
  </button>
</div>
`;
